scripts/pc2validate_interactive.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
```

Comment:
I think the name of this script (`pc2validate_interactive.sh`) is a bit misleading.  The name makes it sound like this is a pc2 script which does "interactive validation".  In fact, I think what this script is doing (correct me if I'm wrong) is running a pc2 "post-validation" step.  That is, the thing known (in CLICS terminology) as the "interactive validator" -- the thing which interacts with the contestant submission and ultimately returns either 42 (for a correct submission) or 43 (for an incorrect submission) --  has _already been run_ when it comes time to run this script, and what this script is doing is handling the things that PC2 needs to do _after the "interactive validator" has been run_.  I think it would be better if the name of the script gave a clearer indication of this -- for example, if it was called `pc2_postinteractive_validation` (or maybe there's a better variant) -- but I think as it's current named, it's confusing about what it's really doing.

Reply:
1) Users should never have to come in contact with this script.  It is internally called by PC2 and never by a user.  For nosy users, there is a nice, verbose comment in the script that describes what it does.  I originally was going to call it: PCVI.sh but I figured that would raise more than 2 eyebrows... (not to mention the confusion with "vi")
2) I thought the name told what it did:  It performs a PC2 style validation of the interactive results: PC2validate_interactive. 
3) Aside from the name being wrong in the usage comment (which I will fix), unless there are major objections, IMHO it can stay as is.  Again, this is a private to PC2 script - no user serviceable parts inside.

Reply:
R.E. Point 1:  note that I didn't say anything about "users" in my initial comment; I was talking about helping _developers_ keep things straight.  

R.E. Point 2:   I guess my argument would be that "PC2 style validation" is itself an ambiguous term.  We have a "PC2 Internal Validator" and a "CLICS Validator"; does the former do "pc2 style validation" and the latter does not?  The question is, what is the "validator"?   Generally, the contest configuration defines (selects, specifies) a "validator" and PC2 _executes_ that validator after each submission's execution against a given test case.   Now, we have a different concept:  PC2 is going to "execute the 'validator' _in order to generate the test case output_, and then is going to _subsequently_ invoke some (new) code, after the "validator" is run, to check the results.  That's why I think calling it "PC2validate_interactive" is misleading: the script does not contain the "validator" code which gets executed (in this case, to generate test case output); the script gets executed _after_ the "validator" gets run.  If we're going to require that the user provide a program, called a "validator", which we will run, then we shouldn't (IMHO) call the script which runs _after_ the validator by a name which sounds (to me) like it is actually a validator.  Maybe a better name would be something like `PC2_interactive_results_validator` or `PC2_post_interactive`.  However, see my next response.

R.E. Point 3: I don't think it's significant that it's a private script; all of the source code is "private" (to the developers).  But if you really think the name should remain as is, I'll accept your decision.

Reply:
Fixed Usage message in comments to contain correct name in next commit.

Point 1 and 3 go hand-in-hand.  I do not consider these scripts source code.  These files are part of the standard distribution in the folder scripts.  We do not include 'java' source code with a distribution, but we do include these.  The fact these are in a separate folder (scripts) and not (bin) means they are none of the user's business to be monkeying around with (this will be in the administrators guide).  These are more "support" files, and i toyed with the idea of stuffing them in the support folder, but there is other seemingly unused stuff in there.

Point 2 is philosophical.  Probably better for a meeting discussion.